src/async_wrap.cc

```diff
@@ -278,7 +278,7 @@ void AsyncWrap::PushAsyncContext(const FunctionCallbackInfo<Value>& args) {
   // then the checks in push_async_ids() and pop_async_id() will.
   double async_id = args[0]->NumberValue(env->context()).FromJust();
   double trigger_async_id = args[1]->NumberValue(env->context()).FromJust();
-  env->async_hooks()->push_async_context(async_id, trigger_async_id, {});
+  env->async_hooks()->push_async_context(async_id, trigger_async_id, nullptr);
```

Comment:
Does this fit to the `CHECK_IMPLIES(resource != nullptr, !resource->IsEmpty())` in `AsyncHooks::push_async_context`?

Reply:
Yes, this is the only call site that passes `nullptr`